src/fides/api/schemas/tcf.py

```diff
@@ -29,7 +29,13 @@ class NonVendorSection(UserSpecificConsentDetails):
     systems: List[EmbeddedVendor] = []  # Systems that use this TCF attribute
 
 
-class TCFPurposeConsentRecord(NonVendorSection, MappedPurpose):
+class CommonPurposeFields(FidesSchema):
```

Comment:
nice, i think this is a good abstraction! we may end up adding more fields to it as we go...